core/src/main/java/com/sequenceiq/cloudbreak/service/organization/OrganizationService.java

```diff
@@ -281,10 +281,13 @@ public Organization delete(Long id) {
     }
 
     public Organization deleteByName(String orgName) {
+        // TODO: check permissions (does the user have the right to delete this org?)
         try {
             return transactionService.required(() -> {
                 Organization organization = getOrganizationOrThrowNotFound(orgName);
+                Long deleted = userOrgPermissionsRepository.deleteByOrganization(organization);
```

Comment:
If there are any resources referencing this organization, probably there will be a constraint violation exception

Reply:
that case should be handled if happens

Reply:
Right that needs to be fixed, and another thing like deleteing you own default org should not be allowed. I will raise a bug for this issue

Reply:
Alright.